### DISCUSSION TITLE
Best Practice for using the model on new videos?

### DISCUSSION THREAD

**Discussion by Brandon-Fricker** [talmolab/sleap#824] [Help!]
Hello! Thank you in advance for your help. 

I've created a model off of pilot videos that I (think) does pretty ok. What is the best practice for using this model on other videos? Just adding them to the project? Or do I create a new project and somehow import the model? I appreciate your help understanding this portion of using SLEAP!

**Comment by roomrys** [comment]
Hi @Brandon-Fricker,

You can run inference on other videos using the above model by either calling the [`sleap-track`](https://sleap.ai/guides/cli.html?%20track#sleap-track) command:
```
sleap-track <path to video> -m <path to model>
```
>Note: if using a top-down model (which uses two models: "centroid" and "centered instance"), you need to specify the second model with an additional `-m <path to model 2>`.

or using the dropdown in the Inference Pipeline GUI to "Select training config file..." as seen below:
![image](https://user-images.githubusercontent.com/38435167/177653596-b099ca1b-9fa1-4e94-8548-05b9924a9624.png)
**Fig: Training Config pipeline with option to "Select training config file". You must select a trained model to run inference.**

However, a [word of caution](https://github.com/talmolab/sleap/discussions/802#discussioncomment-2995711), when training a new model, SLEAP will only use the user-labeled frames in the current SLEAP project to train. To build off of your previous model, you will need to merge the labeled frames into your new SLEAP project.

Thanks,
Liezl
